src/main/kotlin/app/revanced/patches/ecmwf/misc/subscription/fingerprints/SubscriptionUnlockFingerprint.kt

```diff
@@ -0,0 +1,24 @@
+package app.revanced.patches.ecmwf.misc.subscription.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patches.ecmwf.misc.subscription.annotations.SubscriptionUnlockCompatibility
+
+@Name("subscription-unlock")
+@MatchingMethod(
+    "Lcom/garzotto/pflotsh/library_a/MapsActivity", "t0"
+)
+@SubscriptionUnlockCompatibility
+@Version("0.0.1")
+object SubscriptionUnlockFingerprint : MethodFingerprint(
+    "Z",
+    null,
+    null,
+    null,
+    null,
+    { methodDef ->
+        methodDef.definingClass.endsWith("MapsActivity;") && methodDef.name == "t0"
```

Comment:
You constrained the patch compatibility to one version. Hence `t0` would always work, but you should rather make this dynamic with a proper pattern or different fingerprint so the patch works with future versions too.

Reply:
Of course I will make the fingerprint in the future more robust, since the app dont get often updates (1-2 per year) it sould be fine for now.